packages/components/select/src/useSelect.ts

```diff
@@ -225,6 +225,14 @@ export const useSelect = (props, states: States, ctx) => {
     () => states.visible,
     (val) => {
       if (!val) {
+        if (props.filterable) {
+          if (typeof props.filterMethod === 'function') {
```

Comment:
```suggestion
          if (isFunction(props.filterMethod)) {
```

Maybe use the `isFunction` functions from @element-plus/utils